input.py

```diff
@@ -0,0 +1,41 @@
+import os, glob, shutil, requests
+cwd = os.getcwd()
+# Put your kaggle.json in the same folder as input.py
+os.environ["KAGGLE_CONFIG_DIR"] = cwd+"/"
+from kaggle.api.kaggle_api_extended import KaggleApi
+
+api = KaggleApi()
+api.authenticate()
```

Comment:
Warning (Your Kaggle API key is readable by other users on this system) occurs, but this code still works.